data/scripts/template.py

```diff
@@ -119,18 +119,48 @@ def try_subst(verbatim):
 
     return verbatim.replace("@","")
 
-def run_template(f, context):
-    raw = f.read()
+def try_include(verbatim, globals, context):
+    p = re.compile("^.*include.*\"(?P<include_file>.*)\".*$")
+    m = p.match(verbatim)
+
+    if not m:
+        return None, False
+
+    include_file = m.group("include_file")
+    if not include_file:
+        print("Could not determine include_file to include: %s" % verbatim)
+        return None, True
+
+    dir = os.path.dirname(os.path.realpath(args.template.name));
+    path = "%s/%s" % (dir, include_file)
+
+    try:
+        f = open(path)
+    except FileNotFoundError:
```

Comment:
Maybe only capturing `FileNotFoundError` is not enough: one might not have permission to open a file, for instance.